### PR TITLE
Misc changes around segment client

### PR DESCRIPTION
**Pull request by praveenkumar**
**Fixes:** Issue #N

**Relates to:** Issue #N, PR #N, ...

## Solution/Idea

Describe in plain English what you solved and how. For instance, _Added `start` command to CRC so the user can create a VM and set-up a single-node OpenShift cluster on it with one command. It requires blablabla..._

## Proposed changes

List main as well as consequential changes you introduced or had to introduce.

1. Add `start` command.
2. Add `setup` as prerequisite to `start`.
3. ...

## Testing

What is the _bottom-line_ functionality that needs testing? Describe in pseudo-code or in English. Use verifiable statements that tie your changes to existing functionality.

1. `start` succeeds first time after `setup` succeeded
2. stdout contains ... if `start` succeeded
3. stderr contains ... after `start` failed
4. `status` returns ... if `start` succeeded
5. `status` returns ... if `start` failed
6. `start` fails after `start` succeeded or after `status` says CRC is `Running`
7. ...

### CODE REVIEW

File: pkg/os/exec.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 
 func runCmd(command string, args []string, env map[string]string) (string, string, error) {
 	cmd := exec.Command(command, args...) // #nosec G204
+	logging.Debugf("Running '%s %s'", command, strings.Join(args, " "))
```
Comment (cfergeau) [inline pkg/os/exec.go]: nack, `runPrivate` also calls `runCmd`, and we do not want to print the command when `runPrivate` is used.

Comment (praveenkumar) [reply pkg/os/exec.go]: Good catch, will try to change it like we do for oc, send the private flag to run command.

**Review by cfergeau — COMMENTED** [review]
`Your preference will be part of 'crc config set consent-telemetry <yes/no>'`

I find this odd to read. Maybe `Your preference can be changed manually if desired using ``crc config set consent-telemetry <yes/no>`` ` ?

File: pkg/crc/segment/segment.go [hunk 2]
```diff
@@ -34,6 +35,13 @@ func newCustomClient(config *crcConfig.Config, telemetryFilePath, segmentEndpoin
 	client, err := analytics.NewWithConfig(WriteKey, analytics.Config{
 		Endpoint: segmentEndpoint,
 		Logger:   &loggingAdapter{},
+		DefaultContext: &analytics.Context{
+			IP: net.IPv4(0, 0, 0, 0),
+			Library: analytics.LibraryInfo{
```
Comment (guillaumerose) [inline pkg/crc/segment/segment.go:40]: You don't need to set the library. It is done automatically. See https://github.com/segmentio/analytics-go/blob/v3.0/config.go#L162

Comment (praveenkumar) [reply pkg/crc/segment/segment.go:40]: Ah yes, I did check that code but I thought it override whatever in DefaultContext but it just add the `Library` part.

File: pkg/crc/segment/segment.go [hunk 3]
```diff
@@ -34,6 +35,9 @@ func newCustomClient(config *crcConfig.Config, telemetryFilePath, segmentEndpoin
 	client, err := analytics.NewWithConfig(WriteKey, analytics.Config{
 		Endpoint: segmentEndpoint,
 		Logger:   &loggingAdapter{},
+		DefaultContext: &analytics.Context{
+			IP: net.IPv4(0, 0, 0, 0),
```
Comment (anjannath) [inline pkg/crc/segment/segment.go:39]: 👍🏽

File: pkg/crc/preflight/preflight_linux.go [hunk 4]
```diff
@@ -121,7 +121,7 @@ func checkVsock() error {
 	if err != nil {
 		return err
 	}
-	if !strings.Contains(string(getcap), "cap_net_bind_service+eip") {
+	if !strings.Contains(getcap, "cap_net_bind_service+eip") {
```
Comment (anjannath) [inline pkg/crc/preflight/preflight_linux.go:124]: 👍🏽

**Review by anjannath — APPROVED** [review]
LGTM

**Pull request merged**